src/enums/Breed.java

```diff
@@ -0,0 +1,5 @@
+package enums;
+
+public enum Breed {
+    BRITISH, MAINE_COON, PERSIAN, SIAMESE, ORIENTAL
```

Comment:
It is a common practice, to place each value on separate row

Like:
BRITISH,
MAIN_COON, 
etc